schedule/ha/bv/s390x_cluster.yaml

```diff
@@ -0,0 +1,99 @@
+---
+name: s390x_cluster.yaml
+description: >
+  S390x cluster Test. Schedule for the zalpha cluster.
+
+  Currently, SLES+HA tests on s390x are run using the svirt backend (zKVM),
+  where the  openqa workers control via virsh commands, SSH and VNC, a KVM guest
+  on an s390x LPAR host; since networking is provided by the LPAR itself, there
+  is no current way to deploy a Support Server in this scenario, as such,
+  network services are provided outside of the openqa infrastructure.
+
+  Some settings are required in the job group or test suite for this schedule to
+  work.
+
+  The other settings required in the job group are.
+
+  CLUSTER_NAME must be defined for all jobs as a string. HA_CLUSTER_INIT must be
+  defined to a non false value in the parent job HA_CLUSTER_JOIN must be defined
+  for the rest of the jobs, and it must contain the hostname of the parent job
+  (the job where HA_CLUSTER_SETUP is defined to init) HOSTNAME must be defined
+  to different hostnames for each node. All jobs with the exception of the
+  parent job must include a PARALLEL_WITH setting referencing the parent job.
+  HA_UNICAST must be set to 1 because we can not configure multicast corosync on
+  s390x. ISCSI_LUN_INDEX must be set to an available LUN Id starting point,  HA
+  tests will take 5 consecutive LUNs; 1 for SBD, 2 for cluster_md and 2 for
+  drbd_passive. Be very careful with that settings because you may erase LUNs
+  used by other tests. More information here,
+  https://gitlab.suse.de/hsehic/qa-css-docs/-/blob/master/ha/openqa.md#information-specific-to-s390x-tests
+  ISCSI_SERVER must be set to the iscsi server ip address. And of course,
+  NICTYPE and WORKER_CLASS must be set to 'tap' in the job group directly in qemu based jobs.
```

Comment:
`NICTYPE=tap` and `WORKER_CLASS=tap` settings are not needed in `svirt`

Reply:
Good point thx